pkg/cmd/openshift-service-serving-cert-signer/controller.go

```diff
@@ -0,0 +1,74 @@
+package openshift_service_serving_cert_signer
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/origin/pkg/cmd/openshift-service-serving-cert-signer/apis/serviceservingcertsigner/v1alpha1"
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+	servingcertcontroller "github.com/openshift/origin/pkg/service/controller/servingcert"
+)
+
+func RunOpenShiftSSCS(config *v1alpha1.ServiceServingCertSignerConfig, clientConfig *rest.Config, stop <-chan struct{}) error {
+	kubeExternal, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+	defaultInformerResyncPeriod := 10 * time.Minute
```

Comment:
Magic number - make const at top and add comment about why it's a reasonable default.